File: app/src/main/java/org/mifos/mobile/ui/views/UpdatePasswordView.java
package org.mifos.mobile.ui.views;

/*
 * Created by saksham on 13/July/2018
 */

import org.mifos.mobile.ui.views.base.MVPView;

public interface UpdatePasswordView extends MVPView {

    void showError(String message);

    void showPasswordUpdatedSuccessfully();
}

File: app/src/main/java/org/mifos/mobile/ui/views/UpdatePasswordView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.ui.views.base.MVPView

/*
* Created by saksham on 13/July/2018
*/   interface UpdatePasswordView : MVPView {
    fun showError(message: String?)
    fun showPasswordUpdatedSuccessfully()
}